repository/kafka/docs/v0.2/configuration.md

```diff
@@ -0,0 +1,70 @@
+# Configuration 
+
+### Resources
+
+By default, KUDO Kafka resource configuration is set to the minimum recommended values for production usage. 
+But users can and should tune the configurations based on the workload requirements of their Kafka cluster.  
+
+##### Tuning the resources for the Kafka Cluster
+
+```
+kubectl kudo install kafka --instance=my-kafka-name \
+            -p ZOOKEEPER_URI=zk-zookeeper-0.zk-hs:2181,zk-zookeeper-1.zk-hs:2181,zk-zookeeper-2.zk-hs:2181 \
+            -p ZOOKEEPER_PATH=/custom-path \
```

Comment:
```suggestion
            -p ZOOKEEPER_PATH=/<zk_instance_path> \
```

Reply:
not instance path but zk node for kafka cluster. In DC/OS, for example, its `/dcos-service-kafka` if service name is `kafka`. ` /prod/my-kafka-cluster` becomes a zk node in path `/dcos-service-prod__my-kafka-cluster`.

Same principal applies here. It uses KUDO kafka instance name for zk node path. So having a static default can lead to issues.